update-method/src/main/java/com/iluwatar/update/method/Entity.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * The MIT License
+ * Copyright (c) 2014-2016 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.iluwatar.update.method;
+
+/**
+ * @author aungor
+ *
+ */
+public abstract class Entity implements java.io.Closeable{
+  public Entity()
+  {
+	  this.x_ = 0;
+	  this.y_ = 0;
+  }
+
+  public void close()
+  {
+  }
+  public abstract void update();
+
+  public final double x()
+  {
+	  return x_;
+  }
+
+  public final double y()
+  {
+	  return y_;
+  }
+
+  public final void setX(double x)
+  {
+	  x_ = x;
+  }
+  public final void setY(double y)
+  {
+	  y_ = y;
+  }
+
+  private double x_;
+  private double y_;
```

Comment:
The variable naming convention looks odd. Also the program has Checkstyle validation errors that should be corrected.